packages/docs/integrate/extending-integrations.md

```diff
@@ -48,18 +48,66 @@ To register an extension, add it to the array returned from the `extensions` fun
 ```js
 module.exports = {
   integrations: {
-    <TAG NAME>: {
-      location: '@<integration-package>/server',
+    '{INTEGRATION_NAME}': {
+      location: '{SERVER_INTEGRATION}',
```

Comment:
can we add a comment saying what `{SERVER_INTEGRATION}` is? We describe INTEGRATION_NAME later, but SERVER_INTEGRATION is never mentioned.